diffusion_with_autoscaler/autoscaler.py

```diff
@@ -506,7 +506,11 @@ class AutoScaler(LightningFlow):
         scale_in_interval: The number of seconds to wait before checking whether to decrease the number of servers.
         endpoint: Provide the REST API path.
         max_batch_size: (auto-batching) The number of requests to process at once.
-        timeout_batching: (auto-batching) The number of seconds to wait before sending the requests to process.
+        max_batch_delay: (auto-batching) The number of seconds to wait before sending the requests to workers.
+        request_timeout: The number of seconds to wait before timing out a request. A request may timeout because of
```

Comment:
Did you add this already ? Should this be dynamic on the number of element in the batch ?